tests/test_xlsx_to_xml.py

```diff
@@ -3,8 +3,10 @@
 from unittest import TestCase
 import xml.etree.ElementTree as ET
 from unittest.mock import patch, Mock
+from xml.etree.ElementTree import ElementTree
 
-from eva_submission.ENA_submission.xlsx_to_ENA_xml import EnaXlsxConverter
+from eva_submission.ENA_submission.xlsx_to_ENA_xml import EnaXlsxConverter, prettify
```

Comment:
```suggestion
from eva_submission.ENA_submission.xlsx_to_ENA_xml import EnaXlsxConverter
```
(don't think this is being used either)